kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3Storage.java

```diff
@@ -166,7 +169,7 @@ public boolean exists(String name) {
   }
 
   public boolean bucketExists() {
-    return StringUtils.isNotBlank(bucketName) && s3.doesBucketExist(bucketName);
+    return StringUtils.isNotBlank(bucketName) && s3.doesBucketExistV2(bucketName);
```

Comment:
any preference for V2 over non V2?

Reply:
`doesBucketExist` is deprecated and shows up as a warning. 

Details in the AWS SDK about "V1" not being able to distinguish between bucket permissions and wrong credentials. 

https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Client.html#doesBucketExist-java.lang.String-